lecture/admin.py

```
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from .models import Lecture, LectureTitle, LecturePDF
+
+class LectureAdmin(admin.ModelAdmin):
+    list_display = ('get_title', 'classroom', 'uploaded_on', 'slug')  # Display the title from LectureTitle
+    search_fields = ('title__title', 'classroom__name')  # Search by title (from LectureTitle) and classroom
+    list_filter = ('classroom', 'uploaded_on')  # Filters for the list view
+    prepopulated_fields = {'slug': ('title',)}  # Slug will be auto-generated from the LectureTitle
+
+    def get_title(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_title`

(ANN201)